SDK_dotnet/Clients/GluwaClient.cs

```diff
@@ -392,6 +426,100 @@ private string getTimestampSignature(string privateKey)
+                new ABIValue("address", target),
+                new ABIValue("uint256", convertAmount),
+                new ABIValue("uint256", convertFee),
+                new ABIValue("uint256", int.Parse(nonce))
+                );
+
+            EthereumMessageSigner signer = new EthereumMessageSigner();
+            string signature = signer.Sign(messageHash, privateKey);
+
+            return signature;
+        }
+
+        private async Task<string> getBtcSignatureAsync(ECurrency currency, string address, string amount, string fee, string target, string privateKey)
+        {
+            Result<BalanceResponse, ErrorResponse> getUspentOutput = await GetBalanceAsync(currency, address, true);
+            List<UnspentOutput> unspentOutputs = getUspentOutput.Data.UnspentOutputs.OrderByDescending(u => u.Amount).ToList();
+
+            Money amountValue = Money.Parse(amount);
+            Money feeValue = Money.Parse(fee);
+            Money totalAmountAndFeeValue = amountValue + feeValue;
+            BigInteger totalAmountAndFee = new BigInteger(totalAmountAndFeeValue.Satoshi);
+
+            BitcoinAddress sourceAddress = BitcoinAddress.Create(address, mNetwork);
+            BitcoinAddress targetAddress = BitcoinAddress.Create(target, mNetwork);
+            BitcoinSecret secret = new BitcoinSecret(privateKey, mNetwork);
+
+            List<UnspentOutput> usingUnspentOutputs = new List<UnspentOutput>();
+            BigInteger unspentOutputTotalAmount = BigInteger.Zero;
+            for (int i = 0; i < unspentOutputs.Count; i++)
+            {
+                if (totalAmountAndFee > unspentOutputTotalAmount)
+                {
+                    usingUnspentOutputs.Add(unspentOutputs[i]);
+                    Money sumAmount = Money.Parse(unspentOutputs[i].Amount);
+                    unspentOutputTotalAmount += new BigInteger(sumAmount.Satoshi);
+                }
+                else
+                {
+                    break;
+                }
+            }
+
+            List<Coin> conis = new List<Coin>();
+            for (int i = 0; i < usingUnspentOutputs.Count; i++)
+            {
+                conis.Add(new Coin(
+                    fromTxHash: new uint256(usingUnspentOutputs[i].TxHash),
+                    fromOutputIndex: (uint)usingUnspentOutputs[i].Index,
+                    amount: usingUnspentOutputs[i].Amount,
+                    scriptPubKey: Script.FromHex(sourceAddress.ScriptPubKey.ToHex())
+                ));
+            }
+
+            TransactionBuilder builder = mNetwork.CreateTransactionBuilder();
+            NBitcoin.Transaction txn = builder
+                            .AddKeys(secret)
+                            .AddCoins(conis)
+                            .Send(targetAddress, amount)
+                            .SetChange(sourceAddress)
+                            .SendFees(fee)
+                            .BuildTransaction(true);
+
```

Comment:
I recommend you verify the transaction that you created here. Use `builder.Verify(txn, out errors)`